### PR TITLE
Task lwdev 5079 move trades operations from lw

### PR DESCRIPTION
**Pull request by nikagamkrelidze**

### CODE REVIEW

File: src/Lykke.Service.Operations.Contracts/CreateLimitOrderCommand.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿namespace Lykke.Service.Operations.Contracts
+{
+    public class CreateLimitOrderCommand
+    {
+        public AssetPairModel AssetPair { get; set; }
+        public double Volume { get; set; }
+        public decimal Price { get; set; }
+        public OrderAction OrderAction { get; set; }
+        public ClientModel Client { get; set; }
+        public GlobalSettingsModel GlobalSettings { get; set; }
```
Comment (starkmsu) [inline src/Lykke.Service.Operations.Contracts/CreateLimitOrderCommand.cs]: GlobalSettingsModel should be moved out from CreateLimitOrderCommand

File: src/Lykke.Service.Operations.Contracts/CreateMarketOrderCommand.cs [hunk 2]
```diff
@@ -0,0 +1,12 @@
+﻿namespace Lykke.Service.Operations.Contracts
+{
+    public class CreateMarketOrderCommand
+    {
+        public string AssetId { get; set; }
+        public AssetPairModel AssetPair { get; set; }
+        public double Volume { get; set; }
+        public OrderAction OrderAction { get; set; }
+        public ClientModel Client { get; set; }        
+        public GlobalSettingsModel GlobalSettings { get; set; }
```
Comment (starkmsu) [inline src/Lykke.Service.Operations.Contracts/CreateMarketOrderCommand.cs]: GlobalSettingsModel should be moved out from CreateMarketOrderCommand

File: src/Lykke.Service.Operations.Core/Domain/AppGlobalSettings.cs [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+        int LowCashOutLimit { get; }
+        bool MarginTradingEnabled { get; }
+        bool CashOutBlocked { get; }
+        bool BtcOperationsDisabled { get; }
+        bool BitcoinBlockchainOperationsDisabled { get; }
+        bool LimitOrdersEnabled { get; }
+        double MarketOrderPriceDeviation { get; }
+        string[] BlockedAssetPairs { get; set; }
+        string OnReviewAssetConditionLayer { get; set; }
+        DateTime? IcoStartDtForWhitelisted { get; set; }
```
Comment (starkmsu) [inline src/Lykke.Service.Operations.Core/Domain/AppGlobalSettings.cs]: Should be renamed to reflect ICO asset, so corresponding ICO can be identified

File: src/Lykke.Service.Operations.Core/Domain/AppGlobalSettings.cs [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+        DateTime? IcoStartDt { get; set; }
```
Comment (starkmsu) [inline src/Lykke.Service.Operations.Core/Domain/AppGlobalSettings.cs]: Same as above

File: src/Lykke.Service.Operations.Core/Domain/AppGlobalSettings.cs [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+        public bool MarginTradingEnabled { get; set; }
+        public bool CashOutBlocked { get; set; }
+        public bool BtcOperationsDisabled { get; set; }
+        public bool BitcoinBlockchainOperationsDisabled { get; set; }
+        public bool LimitOrdersEnabled { get; set; }
+        public double MarketOrderPriceDeviation { get; set; }
+        public string[] BlockedAssetPairs { get; set; }
+        public string OnReviewAssetConditionLayer { get; set; }
+        public DateTime? IcoStartDtForWhitelisted { get; set; }
+        public DateTime? IcoStartDt { get; set; }
```
Comment (starkmsu) [inline src/Lykke.Service.Operations.Core/Domain/AppGlobalSettings.cs]: Same here

File: src/Lykke.Service.Operations.Contracts/AssetModel.cs [hunk 6]
```diff
@@ -0,0 +1,13 @@
+﻿namespace Lykke.Service.Operations.Contracts
+{
+    public class AssetModel
```
Comment (starkmsu) [inline src/Lykke.Service.Operations.Contracts/AssetModel.cs]: Public class should have xml doc

File: src/Lykke.Service.Operations.Contracts/AssetPairModel.cs [hunk 7]
```diff
@@ -0,0 +1,11 @@
+﻿namespace Lykke.Service.Operations.Contracts
+{
+    public class AssetPairModel
```
Comment (starkmsu) [inline src/Lykke.Service.Operations.Contracts/AssetPairModel.cs]: Public class should have xml doc

File: src/Lykke.Service.Operations.Contracts/ClientModel.cs [hunk 8]
```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace Lykke.Service.Operations.Contracts
+{
+    public class ClientModel
```
Comment (starkmsu) [inline src/Lykke.Service.Operations.Contracts/ClientModel.cs]: Public class should have xml doc

File: src/Lykke.Service.Operations.Contracts/CreateLimitOrderCommand.cs [hunk 9]
```diff
@@ -0,0 +1,12 @@
+﻿namespace Lykke.Service.Operations.Contracts
+{
+    public class CreateLimitOrderCommand
```
Comment (starkmsu) [inline src/Lykke.Service.Operations.Contracts/CreateLimitOrderCommand.cs]: Public class should have xml doc

File: src/Lykke.Service.Operations.Contracts/CreateMarketOrderCommand.cs [hunk 10]
```diff
@@ -0,0 +1,12 @@
+﻿namespace Lykke.Service.Operations.Contracts
+{
+    public class CreateMarketOrderCommand
```
Comment (starkmsu) [inline src/Lykke.Service.Operations.Contracts/CreateMarketOrderCommand.cs]: Public class should have xml doc

File: src/Lykke.Service.Operations.Contracts/Events/LimitOrderCreatedEvent.cs [hunk 11]
```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace Lykke.Service.Operations.Contracts.Events
+{
+    public class LimitOrderCreatedEvent
```
Comment (starkmsu) [inline src/Lykke.Service.Operations.Contracts/Events/LimitOrderCreatedEvent.cs]: Public class should have xml doc

File: src/Lykke.Service.Operations.Contracts/Events/LimitOrderRejectedEvent.cs [hunk 12]
```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace Lykke.Service.Operations.Contracts.Events
+{
+    public class LimitOrderRejectedEvent
```
Comment (starkmsu) [inline src/Lykke.Service.Operations.Contracts/Events/LimitOrderRejectedEvent.cs]: Public class should have xml doc

File: src/Lykke.Service.Operations.Contracts/Events/OperationCreatedEvent.cs [hunk 13]
```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace Lykke.Service.Operations.Contracts.Events
+{
+    public class OperationCreatedEvent
```
Comment (starkmsu) [inline src/Lykke.Service.Operations.Contracts/Events/OperationCreatedEvent.cs]: Public class should have xml doc

File: src/Lykke.Service.Operations.Contracts/FeeSettingsModel.cs [hunk 14]
```diff
@@ -0,0 +1,8 @@
+﻿namespace Lykke.Service.Operations.Contracts
+{
+    public class FeeSettingsModel
```
Comment (starkmsu) [inline src/Lykke.Service.Operations.Contracts/FeeSettingsModel.cs]: Public class should have xml doc

File: src/Lykke.Service.Operations.Contracts/GlobalSettingsModel.cs [hunk 15]
```diff
@@ -0,0 +1,11 @@
+﻿namespace Lykke.Service.Operations.Contracts
+{
+    public class GlobalSettingsModel
```
Comment (starkmsu) [inline src/Lykke.Service.Operations.Contracts/GlobalSettingsModel.cs]: Public class should have xml doc

File: src/Lykke.Service.Operations.Contracts/IcoSettingsModel.cs [hunk 16]
```diff
@@ -0,0 +1,8 @@
+﻿namespace Lykke.Service.Operations.Contracts
+{
+    public class IcoSettingsModel
```
Comment (starkmsu) [inline src/Lykke.Service.Operations.Contracts/IcoSettingsModel.cs]: Public class should have xml doc

File: src/Lykke.Service.Operations.Contracts/OperationStatus.cs [hunk 17]
```diff
@@ -0,0 +1,15 @@
+﻿using System;
+
+namespace Lykke.Service.Operations.Contracts
+{
+    [Flags]
+    public enum OperationStatus
```
Comment (starkmsu) [inline src/Lykke.Service.Operations.Contracts/OperationStatus.cs]: Public enum should have xml doc

File: src/Lykke.Service.Operations.Contracts/OperationType.cs [hunk 18]
```diff
@@ -0,0 +1,11 @@
+﻿namespace Lykke.Service.Operations.Contracts
+{
+    public enum OperationType
```
Comment (starkmsu) [inline src/Lykke.Service.Operations.Contracts/OperationType.cs]: Public enum should have xml doc

File: src/Lykke.Service.Operations.Contracts/OrderAction.cs [hunk 19]
```diff
@@ -0,0 +1,7 @@
+﻿namespace Lykke.Service.Operations.Contracts
+{
+    public enum OrderAction
```
Comment (starkmsu) [inline src/Lykke.Service.Operations.Contracts/OrderAction.cs]: Public enum should have xml doc

File: src/Lykke.Service.Operations.Contracts/PersonalDataModel.cs [hunk 20]
```diff
@@ -0,0 +1,9 @@
+﻿namespace Lykke.Service.Operations.Contracts
+{
+    public class PersonalDataModel
```
Comment (starkmsu) [inline src/Lykke.Service.Operations.Contracts/PersonalDataModel.cs]: Public class should have xml doc

File: src/Lykke.Service.Operations.Contracts/WalletModel.cs [hunk 21]
```diff
@@ -0,0 +1,7 @@
+﻿namespace Lykke.Service.Operations.Contracts
+{
+    public class WalletModel
```
Comment (starkmsu) [inline src/Lykke.Service.Operations.Contracts/WalletModel.cs]: Public class should have xml doc

File: src/Lykke.Service.Operations.Core/Repositories/IAppGlobalSettingsRepositry.cs [hunk 22]
```diff
@@ -0,0 +1,23 @@
...
+using System.Threading.Tasks;
+using Lykke.Service.Operations.Core.Domain;
+
+namespace Lykke.Service.Operations.Core.Repositories
+{
+    public interface IAppGlobalSettingsRepositry
+    {
+        Task SaveAsync(IAppGlobalSettings appGlobalSettings);
+
+        Task UpdateAsync(string depositUrl = null, bool? debugMode = null,
```
Comment (starkmsu) [inline src/Lykke.Service.Operations.Core/Repositories/IAppGlobalSettingsRepositry.cs]: Better replace it with a set of Update... methods

File: src/Lykke.Service.Operations/Lykke.Service.Operations.csproj [hunk 23]
```diff
@@ -38,16 +32,26 @@
     <PackageReference Include="AutoMapper" Version="6.2.2" />
     <PackageReference Include="FluentValidation" Version="7.4.0" />
     <PackageReference Include="FluentValidation.AspNetCore" Version="7.4.0" />
-    <PackageReference Include="Lykke.Common" Version="4.4.2" />
+    <PackageReference Include="Lykke.Common" Version="6.1.0" />
     <PackageReference Include="Lykke.Common.ApiLibrary" Version="1.3.1" />
+    <PackageReference Include="Lykke.Cqrs" Version="4.4.0" />
     <PackageReference Include="Lykke.Logs" Version="3.6.0" />
```
Comment (starkmsu) [inline src/Lykke.Service.Operations/Lykke.Service.Operations.csproj]: It's better to update Lykke.Logs to the latest version

File: src/Lykke.Service.Operations.Contracts/CreateMarketOrderCommand.cs (hunk 2 above)
Comment (mmusin) [reply src/Lykke.Service.Operations.Contracts/CreateMarketOrderCommand.cs]: ?

File: src/Lykke.Service.Operations.Contracts/CreateLimitOrderCommand.cs (hunk 1 above)
Comment (mmusin) [reply src/Lykke.Service.Operations.Contracts/CreateLimitOrderCommand.cs]: It’s part of the command context

**Pull request merged**